tests/trial_tests/test_trials.py

```diff
@@ -0,0 +1,233 @@
+import datetime
+from typing import Any
+from typing import Dict
+from typing import Optional
+
+import pytest
+
+from optuna.distributions import BaseDistribution
+from optuna.distributions import CategoricalDistribution
+from optuna.distributions import FloatDistribution
+from optuna.distributions import IntDistribution
+from optuna.study.study import create_study
+from optuna.trial import BaseTrial
+from optuna.trial import create_trial
+from optuna.trial import FixedTrial
+from optuna.trial import FrozenTrial
+from optuna.trial import Trial
+
+
+parametrize_trial_type = pytest.mark.parametrize("trial_type", [FixedTrial, FrozenTrial])
```

Comment:
I feel uncomfortable that Trial is not included here, why not use parametrize_trial_type in tests where Trial cannot be included (e.g. test_suggest_float)?

Reply:
Sorry, could you elaborate on that?